src/lib/containers/CertificationQuiz.tsx

```diff
@@ -0,0 +1,210 @@
+import React, { useEffect, useState } from 'react'
+import { SynapseClient } from '../utils'
+import { useSynapseContext } from '../utils/SynapseContext'
+import {
+  PassingRecord,
+  QuizResponse,
+  Quiz,
+} from '../utils/synapseTypes/CertificationQuiz/Quiz'
+import {
+  MULTICHOICE_RESPONSE_CONCRETE_TYPE_VALUE,
+  QuestionResponse,
+} from '../utils/synapseTypes/CertificationQuiz/Questions'
+import parse from 'html-react-parser'
+import { displayToast } from './ToastMessage'
+import { Button } from 'react-bootstrap'
+import { MarkdownPopover } from './MarkdownPopover'
+import { HelpOutlineTwoTone } from '@material-ui/icons'
+import Typography from '../utils/typography/Typography'
+
+const CertificationQuiz: React.FunctionComponent = () => {
+  const { accessToken } = useSynapseContext()
+  const [quiz, setQuiz] = useState<Quiz | undefined>()
+  const [questionResponse, setQuestionResponse] = useState<QuestionResponse[]>(
+    [],
+  )
+  const [passingRecord, setPassingRecord] = useState<PassingRecord>()
+
+  const gettingStartedUrl =
+    'https://help.synapse.org/docs/Getting-Started.2055471150.html'
+
+  const getQuiz = async () => {
+    try {
+      setQuiz(await SynapseClient.getCertifyQuiz(accessToken))
+    } catch (err: any) {
+      console.log(err)
+    }
+  }
+
+  useEffect(() => {
+    if (accessToken) {
+      getQuiz()
+    }
+  }, [accessToken])
```

Comment:
IMO all of the state/effects are cleaner if we used `react-query`, but I recognize that you would have to manually configure the options and manipulate the cache afterwards so the quiz doesn't get overwritten. So I'll leave it up to you whether to keep what you have or make the change.

```tsx
const { data: quiz } = useGetCertifyQuiz({
  enabled: !!accessToken,
  staleTime: Infinity,
  cacheTime: Infinity
})
```